tests/ui/macros/format-issue-130170.rs

```diff
@@ -0,0 +1,11 @@
+//@ run-rustfix
+
+fn main() {
+    let s = "123";
+    println!({}, "sss", s);
```

Comment:
Suggestion: if this is intended to provide the suggestion for both `println!({}, "sss", s)` and `println!((), "sss", s)` cases, please add a test case for both.

Reply:
> Suggestion: if this is intended to provide the suggestion for both `println!({}, "sss", s)` and `println!((), "sss", s)` cases, please add a test case for both.

Done. Thank you.